neuroanalysis/fitting/exp.py

```diff
@@ -24,8 +24,8 @@ def estimate_exp_params(data):
     params : tuple
         (yoffset, yscale, tau, toffset)
     """
-    start_y = data.data[:len(data.data)//100].mean()
-    end_y = data.data[-len(data.data)//10:].mean()
+    start_y = data.data[:len(data.data)//10].mean()
+    end_y = data.data[-len(data.data)//100:].mean()
```

Comment:
Why these?

Reply:
the changes at the beginning of the expected curve are more drastic than the changes at the end. thus, the last hundred samples are likely going to be the same, while the first hundred samples are not.

ooooh, wait, I just read the math. it's not count of samples, it's "tenth of all samples" v. "hundredth of all samples". I read it backwards.